### PR TITLE
ADEN-4275 Quick fix: ignore user click

### PR DESCRIPTION
**Pull request by v3rth**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/video/googleIma.js [hunk 1]
```diff
@@ -4,10 +4,11 @@ define('ext.wikia.adEngine.video.googleIma', [
 	'ext.wikia.adEngine.video.googleImaAdStatus',
 	'ext.wikia.adEngine.video.player.ui.closeButton',
 	'ext.wikia.adEngine.video.volumeControlHandler',
+	'wikia.browserDetect',
 	'wikia.document',
 	'wikia.log',
 	'wikia.window'
-], function (scriptLoader, googleImaAdStatus, closeButton, volumeControlHandler, doc, log, win) {
+], function (scriptLoader, googleImaAdStatus, closeButton, volumeControlHandler, browserDetect, doc, log, win) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/video/googleIma.js:11]: MAJOR Function has 8 parameters which is greater than 7 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* MAJOR 1 major

Watch the comments in this conversation to review them.

File: extensions/wikia/AdEngine/js/video/googleIma.js [hunk 2]
```diff
@@ -127,12 +129,16 @@ define('ext.wikia.adEngine.video.googleIma', [
 						self.adDisplayContainer.initialize();
 						self.adsManager.init(width, height, google.ima.ViewMode.NORMAL);
 						self.adsManager.start();
+						log('Video play: stared', log.levels.debug, logGroup);
```
Comment (drets) [inline extensions/wikia/AdEngine/js/video/googleIma.js]: started?

File: extensions/wikia/AdEngine/js/video/googleIma.js [hunk 3]
```diff
@@ -127,12 +129,16 @@ define('ext.wikia.adEngine.video.googleIma', [
...
 					};
 
 				if (this.isAdsManagerLoaded) {
 					callback();
-				} else {
+				} else if (!browserDetect.isMobile()) { // ADEN-4275 quick fix
+					log(['Video play: waiting for full load of adsManager'], log.levels.debug, logGroup);
 					this.adsLoader.addEventListener(google.ima.AdsManagerLoadedEvent.Type.ADS_MANAGER_LOADED, callback, false);
+				} else {
+					log(['Video play: trigger video play action is ignored'], log.levels.warning, logGroup);
```
Comment (drets) [inline extensions/wikia/AdEngine/js/video/googleIma.js:141]: Do we want to ignore the user click on iPad (`browserDetect.isMobile() === true`) as well (where we have Oasis skin)?

**Review by bognix — APPROVED** [review]

File: extensions/wikia/AdEngine/js/video/googleIma.js (hunk 3 above)
Comment (v3rth) [reply extensions/wikia/AdEngine/js/video/googleIma.js:141]: Right now I don't know how browser on iPads behaves, but I guess same as for mobile devices (throw errors) what means we want to ignore it

Comment (drets) [reply extensions/wikia/AdEngine/js/video/googleIma.js:141]: :ok_hand:  thanks, by this question I wanted _kindly_ ask you to check.

**Pull request merged**